cmd/version.go

```diff
@@ -44,6 +45,12 @@ type version struct {
 
 // versionString returns the CLI version
 func versionString() string {
+	if kubeBuilderVersion == "" {
```

Comment:
It seems to this code will never calls, as by default kubeBuilderVersion is always `unknown`.
You should introduce a new constant with "unknown" value, for example
```golang 
const unknown = "unknown"

var (
	kubeBuilderVersion      = unknown
	kubernetesVendorVersion = unknown
	goos                    = unknown
	goarch                  = unknown
	...
)
```
and compare here with it instead of empty string.

Reply:
@migueleliasweb  this code should be changed also
```
const unknown = "unknown"

var (
	kubeBuilderVersion      = unknown
	kubernetesVendorVersion = unknown
	goos                    = unknown
	goarch                  = unknown
	...
)
```

Reply:
Fixed changes. I was trying to find your comment but Github's UI wasn't helping it much. Found it a few minutes later.